server/schedulers/hot_region.go

```diff
@@ -617,7 +617,9 @@ func (bs *balanceSolver) filterHotPeers() []*statistics.HotPeerStat {
 
 	// filter pending region
 	appendItem := func(items []*statistics.HotPeerStat, item *statistics.HotPeerStat) []*statistics.HotPeerStat {
-		if _, ok := bs.sche.regionPendings[item.ID()]; !ok && !item.IsJustTransferLeader() {
+		minHotDegree := bs.cluster.GetOpts().GetHotRegionCacheHitsThreshold()
+		if _, ok := bs.sche.regionPendings[item.ID()]; !ok && !item.IsNeedCoolDownTransferLeader(minHotDegree) {
```

Comment:
Will it affect the move peer operation?

Reply:
Neither the transfer leader operator nor the move peer operator will be generated within three minutes of the transfer leader